### PR TITLE
Student teacher fix

### PR DESCRIPTION
**Pull request by ravimariya**

### CODE REVIEW

File: app/controllers/students/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   end
+
+   def create
+     super
+   end
+
+  # DELETE /resource/sign_out
+   def destroy
+    super
+    end
```
Comment (HirenBhalani) [inline app/controllers/students/sessions_controller.rb]: Indentation for all files should be same, set indentation of 2 spaces

File: app/controllers/admins_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AdminsController < ApplicationController
+ before_action :authenticate_admin!,only: [:welcome]
+	def welcome
```
Comment (HirenBhalani-BTC) [inline app/controllers/admins_controller.rb]: If there is only one method require before filter then you can set it like
`before_action :authenticate_admin!,only: :welcome`
 no need to mention in array

File: app/controllers/students_controller.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+		@student=Student.new
+	end
+
+	def create
+   @student=Student.new(student_params)
+
+ 
+   if @student.save
+   
+      no=Time.now.year.to_s+@student.standard_id.to_s+("0".*(4-@student.id.to_s.length))+@student.id.to_s
```
Comment (HirenBhalani) [inline app/controllers/students_controller.rb]: What is variable 'no', how can i know purpose of this variable.
variable name should be meaningful 
set appropriate variable name.

File: app/controllers/students_controller.rb [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      	
+        	render :new
+    	
+    end
+
+  end
+	
+  def update
+		if @student.update(student_params)
+    	  	redirect_to list_students_path
```
Comment (HirenBhalani) [inline app/controllers/students_controller.rb]: Indentation should be proper in all files.

File: app/models/student.rb [hunk 5]
```diff
@@ -0,0 +1,33 @@
+class Student < ActiveRecord::Base
+	belongs_to:standard
+	validates:first_name,presence:true
+	validates:middle_name,presence:true
+	validates:last_name,presence:true
+	validates:date_of_birth,presence:true
+	validates:address,presence:true
+	validates:city,presence:true
+	validates:contactno,presence:true,numericality:{only_integer:true}
+	validates:standard_id,presence:true
```
Comment (HirenBhalani) [inline app/models/student.rb]: You can set validations like
validates :first_name, :middle_name, :last_name, ...., presence: true

check all controller and change the code.

File: app/views/layouts/login_layout.html.erb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+  <title>PerformanceAnalyser</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body >
+<div class="container">
```
Comment (HirenBhalani) [inline app/views/layouts/login_layout.html.erb]: Where is this div tag completed?

File: app/views/students/edit.html.erb [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+<div class="col-md-6">
+<div class="login-panel panel panel-default">
+<div class="panel-body">
+ <div class="panel-form">
+<fieldset>
+
+<div class="form-group <%= 'has-error' if @student.errors.messages[:date_of_birth].present? %>">
+  <%= f.label :Date_Of_Birth,class: "col-sm-4"%>
+    <div class="col-sm-8">
+      <%= f.text_field :date_of_birth,class: "form-control datepicker",placeholder: "DD-MM-YYYY" %>
```
Comment (HirenBhalani) [inline app/views/students/edit.html.erb]: add one space after ,(comma).
remove space before ,(comma).

you have to follow ruby style guide.
check all files and change accordingly.

File: app/views/layouts/login_layout.html.erb (hunk 6 above)
Comment (ravimariya) [inline app/views/layouts/login_layout.html.erb]: its fixed now sir

File: app/controllers/students/sessions_controller.rb (hunk 1 above)
Comment (ravimariya) [inline app/controllers/students/sessions_controller.rb]: ok sir ,fixing indentation in all files

File: app/controllers/students_controller.rb (hunk 3 above)
Comment (ravimariya) [inline app/controllers/students_controller.rb]: its changed now to enrolment_no

File: app/views/admins/sessions/new.html.erb [hunk 8]
```diff
@@ -1,18 +1,35 @@
...
+							</div>
+						</fieldset>
+
+						<% end %>
+					</div>
+				</div>
+			</div>
+		</div>                   
+	</div>
+</div>
```
Comment (HirenBhalani) [inline app/views/admins/sessions/new.html.erb:35]: Is this files has 2 space indentation?

File: app/views/students/sessions/new.html.erb [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+					</fieldset>
+
+					<% end %>
+				</div>
+			</div>
+		</div>
+	</div>                   
+</div>
+</div>
+]
```
Comment (HirenBhalani) [inline app/views/students/sessions/new.html.erb]: Why this is in array?

Comment (ravimariya) [inline app/views/students/sessions/new.html.erb]: it was mistake by my side,its removed now

File: app/views/admins/sessions/new.html.erb (hunk 8 above)
Comment (ravimariya) [inline app/views/admins/sessions/new.html.erb:35]: yes sir i set 2 spaces for indentation

**Pull request merged**